x/staking/client/cli/query.go

```diff
@@ -342,7 +342,7 @@ $ %s query staking delegations cosmos1gghjut3ccd8ay0zduzj64hwre2fxs9ld75ru9p
 				return err
 			}
 
-			return clientCtx.PrintOutput(res.DelegationResponses)
+			return clientCtx.PrintOutput(res)
```

Comment:
We should add `cli-tests` for these queries, I remember @sahith-narahari working on querier tests.